projects/packages/masterbar/src/nudges/additional-css/class-atomic-additional-css-manager.php

```diff
@@ -36,8 +36,11 @@ public function __construct( $domain ) {
 	 * @param \WP_Customize_Manager $wp_customize_manager Core customize manager.
 	 */
 	public function register_nudge( \WP_Customize_Manager $wp_customize_manager ) {
+		$plan = \Automattic\Jetpack\Plans::get_plan( 'business_bundle' )->product_name_short;
```

Comment:
Shouldn't we take into consideration if the feature flag is enabled?

Reply:
No. On Atomic sites, we always nudge them to upgrade to Business, not Personal (or Premium).

Reply:
This seems a bit off, why do you nudge to Business instead of the minimum plan?

Reply:
I agree is a bit off, but this is needed because we don't support Personal and Premium plans on Atomic afaik. If they were to upgrade to these plans, the sites would still be reverted to Simple.

In any case, changing this behavior is out of scope for this PR. :)